ui/app/components/status-report-meta-table/index.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="artifact-overview">
```

Comment:
There are a few localizable strings in this file. Could we add them to `en-us.yaml`?